AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -810,20 +825,24 @@ - (void)reloadRowsAtIndexPaths:(NSArray *)indexPaths withAnimationOptions:(ASDat
     
     // Reloading requires re-fetching the data.  Load it on the current calling thread, locking the data source.
     [self accessDataSourceWithBlock:^{
-      NSMutableArray *nodes = [[NSMutableArray alloc] initWithCapacity:indexPaths.count];
+      NSMutableArray<ASIndexedNodeContext *> *contexts = [[NSMutableArray alloc] initWithCapacity:indexPaths.count];
       
       // FIXME: This doesn't currently do anything
```

Comment:
Yeah, `ASChangeSetDataController` was introduced in a way that it's possible to fallback to `ASDataController` and it can still do its job (reasonably well). With recent work on `ASChangeSetDataController`, I guess it is an essential part now and no one should ever allocate `ASDataController` directly. As soon as we can enforce that, we can get rid of all the redundant sorts. Thoughts?